sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/DocumentClassifierDetails.java

```diff
@@ -122,25 +122,25 @@ void setExpirationDateTime(OffsetDateTime expirationDateTime) {
      *
      * @return the apiVersion value.
      */
-    public String getApiVersion() {
-        return this.apiVersion;
+    public String getServiceVersion() {
```

Comment:
From the description, it seems a "service API version", Should we consider aligning the naming to "getServiceApiVersion"?

Reply:
This is arch board feedback and consistent with ServiceVersion model we already have exposed.